subprojects/common/solver-z3/src/test/java/hu/bme/mit/theta/solver/z3/Z3SolverTest.java

```diff
@@ -152,6 +155,37 @@ public void testArray() {
 		assertEquals(Int(2), Read(valLit, Int(1)).eval(ImmutableValuation.empty()));
 	}
 
+	@Test
+	public void testArrayInit() {
+        final ConstDecl<ArrayType<IntType, IntType>> arr = Const("arr", Array(Int(), Int()));
+		var elems = new ArrayList<Tuple2<Expr<IntType>, Expr<IntType>>>();
```

Comment:
Formatting seems to be a bit inconsistent.

Reply:
I don't know how I managed that, because in IntelliJ it looked fine, but I fixed it now.